src/acceptance-test/java/org/zalando/nakadi/webservice/hila/SubscriptionAT.java

```diff
@@ -112,6 +112,44 @@ public void testSubscriptionBaseOperations() throws IOException {
     }
 
     @Test
+    public void testSubscriptionWithManyEventTypesIsCreated() throws IOException {
+        final List<String> eventTypes = IntStream.range(0, 30).mapToObj(i -> createEventType())
+                .map(et -> et.getName())
+                .collect(Collectors.toList());
+        final String subscription = "{\"owning_application\":\"app\",\"event_types\":" +
+                "[" + eventTypes.stream().map(et -> "\"" + et + "\"").collect(Collectors.joining(",")) + "]}";
+        final Response response = given()
+                .body(subscription)
+                .contentType(JSON)
+                .post(SUBSCRIPTIONS_URL);
+        // assert response
+        response.then().statusCode(HttpStatus.SC_CREATED).contentType(JSON);
+        final Subscription gotSubscription = MAPPER.readValue(response.print(), Subscription.class);
+        Assert.assertNotNull(gotSubscription.getId());
+    }
+
+    @Test
+    public void testSubscriptionWithManyEventTypesIsNotCreated() {
```

Comment:
I think these two tests should be unit tests. I see no reason for them to be acceptance tests.